scripts/add_license.sh

```diff
@@ -14,10 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Adds lincense header to files.
-# FIXME: Reserve shebang
+# Adds a lincense header to files.
 
-LICENSE_LINE='Licensed under the Apache License, Version 2.0 (the "License");'
+LICENSE_LINE='Licensed under the Apache License, '
+LICENSE_LINE+='Version 2.0 (the "License");'
 
 LICENSE_HEADER=(
```

Comment:
Can we read this string from another file like `license_header.txt` so the check script can check the full string of the header?